extension/scripts/features/upgrade-required-respect/ttUpgradeRequiredRespect.js

```diff
@@ -35,6 +35,7 @@
 				.trim()
 		);
 		const requiredNode = document.find("#faction-upgrades #stu-confirmation div[role] > :nth-child(3) > .text");

```

Comment:
I'd think changing this to `#faction-upgrades #stu-confirmation div[role] .required .text` would be better. Then return if it's not found.

Reply:
It doesn't work.